src/amo/actions/ratings.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+import { SET_USER_RATING } from 'amo/constants';
+
+export const setUserRating = ({ addonID, userRating }) => ({
+  type: SET_USER_RATING,
+  data: {
+    addonID,
```

Comment:
I think these names with `ID` at the end should have `Id` instead. The only other example we have is `trackingId` but we've been writing `API` suffixes as `fooApi` as well.